components/inputs/test/input-date.vdiff.js

```diff
@@ -351,6 +351,68 @@ describe('d2l-input-date', () => {
 				await sendKeysElem(elem, 'press', 'Enter');
 				await expect(elem).to.be.golden();
 			});
+
+			it('click then click away', async() => {
+				await clickElem(elem);
+				await clickElem(document.body);
+				await expect(elem).to.be.golden();
+			});
+
+			it('open then close', async() => {
+				await sendKeysElem(elem, 'press', 'Enter');
+				sendKeys('press', 'Escape');
+				await oneEvent(elem, 'd2l-tooltip-show');
+				await expect(elem).to.be.golden();
+			});
+		});
+
+		describe('required min-max revert', () => {
+
+			it('delete text input then blur', async() => {
+				const elem = await fixture(create({ label: 'Date', labelHidden: false, required: true, value: '2012-01-01', maxValue: '2018-02-27', minValue: '2018-02-13' }));
+				await focusElem(elem);
+				await sendKeysElem(elem, 'press', 'Tab');
+
+				await focusElem(elem);
+				await sendKeysElem(elem, 'press', 'Backspace');
+				await sendKeys('press', 'Tab');
+				await expect(elem).to.be.golden();
+			});
+
+		});
+
+		describe('required revert', () => {
+
+			let elem;
+			beforeEach(async() => {
+				elem = await fixture(create({ label: 'Date', labelHidden: false, required: true, value: '2020-01-01' }));
+			});
+
+			it('delete text input then blur', async() => {
+				await focusElem(elem);
+				await sendKeysElem(elem, 'press', 'Backspace');
+				await sendKeys('press', 'Tab');
+				await expect(elem).to.be.golden();
+			});
+
+			it('delete text input then blur then re-focus', async() => {
+				await focusElem(elem);
+				await sendKeysElem(elem, 'press', 'Backspace');
+				await sendKeys('press', 'Tab');
+				await focusElem(elem);
+				await aTimeout(100);
```

Comment:
Since the blur instructions and refocus instructions now use the same tooltip, we can no longer wait for the `d2l-tooltip-show` event in order to get the image. I tried a variety of things but the only reliable one I've found is `aTimeout`.